.travis.yml

```diff
@@ -48,6 +48,7 @@ install:
   # etsdemo
   - pushd ets-demo && for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} || exit; done && popd
 script:
+  - for toolkit in ${TOOLKITS}; do edm run -- python etstool.py flake8 --runtime=${RUNTIME} --toolkit=${toolkit} || exit; done
```

Comment:
Initially I thought I would not care about the redundancy in looping over the toolkit. Turns out, this command takes 28s on Travis. If I only run it for one toolkit, it will be 7s. 28s is a bit long for my liking, so I think I will run it for one toolkit only.

Reply:
FWIW, the GitHub actions setup in Traits makes this sort of thing quite nice and easy: you can have multiple separate actions, so for Traits, while the unit tests are run on all 15 platform / Python version combinations, flake8 checks are only run on 2 combinations (Python 3.6 and Python 3.8 on Linux).

Reply:
i would even go as far as saying only run it in one job - not in every single matrix + travis and appveyor but that's a separate discussion.

Reply:
Hmmm, it might be cleaner to actually pull out the null toolkit to a separate job and only run flake8 in that job... again with the ugly inline if.